test/system/src/main/java/io/pravega/test/system/framework/services/docker/BookkeeperDockerService.java

```diff
@@ -80,13 +79,20 @@ private ServiceSpec setServiceSpec() {
         stringList.add(env2);
         stringList.add(env3);
         stringList.add(env4);
+
+        String cmd1 = "CMD-SHELL";
```

Comment:
We should create a common healthCheck method inside `DockerBasedService.java` which can be used by all other services. This could be the default health check for all services.

Reply:
added `commonHealthCheck()`.